src/util/hooks.ts

```diff
@@ -169,3 +172,29 @@ export const useScreenOrientation = () => {
 
     return orientation;
 };
+
+export const useSyncServerImages = (fetchImage: number) => {
```

Comment:
Comment is needed to state under what situation should we want to fetch the images from server. (User imports others work with cloud images -> need fetch them to display)